### DISCUSSION TITLE
削除済のユーザが非公開にしたページを更新する方法

### DISCUSSION THREAD

**Discussion by ghost** [growilabs/growi#9051] [Help / Questions]
ページの公開範囲を「自分のみ」に設定したユーザが削除されると、そのページは二度と更新できなくなるのでしょうか？
lsxのリストやページ数表示に影響が出るため、削除済のユーザが非公開にしたページを削除したいです。

**Comment by yuki-takei** [comment]
現時点では仰る通り他のユーザーから「自分のみ」に設定したページを個別に操作することはできません。

管理画面からそのようなページを一括で削除できるようにしてもいいかもしれません。
もしくは他のアイデアがあるでしょうか

**Reply by ghost** [reply]
少し強引ですが、下記の手順で削除することができました。
なお、他のユーザが「自分のみ」に設定していて編集できないページのことを、ここでは「非公開のページ」と呼ばせていただきます。

①非公開のページと同じ階層に存在するページを他の場所に退避させる
②親ページに移動し、「全ての子ページも削除」にチェックを入れて親ページを削除する
③ゴミ箱に移動し、「全ての子ページも元に戻す」のチェックを外して親ページを元に戻す
④退避させていたページを元の場所に戻す

※削除したくない非公開のページが同じ階層に存在する場合、手順①の退避ができないため使用できません
